src/VersionTool/Program.cs

```diff
@@ -59,9 +60,22 @@ public static void Main(string[] args)
 
                 c.HelpOption("-h|--help");
 
-                c.OnExecute(() => OnUpdateDependency(pathOption, matchingOption, dependencyArgument, versionArgument));
+                c.OnExecute(() => OnExecuteUpdateDependency(pathOption, matchingOption, dependencyArgument, versionArgument));
             });
 
+            var updatePatch = app.Command("update-patch", (c) =>
+            {
+                var pathOption = c.Option("-d|--directory", "Directory containing all repos", CommandOptionType.SingleValue);
+                var updatePatchConfigOption = c.Option("-u|--update-patchconfig", "Update patchconfig with all packages found and their current versions", CommandOptionType.NoValue);
+                var updatePackageOption = c.Option("-p|--update-package", "Update the given package before patch following the format <package>;<version>", CommandOptionType.MultipleValue);
+                var updateRepoOption = c.Option("-r|--update-repo", "Update the packages in the given repo before patch ", CommandOptionType.MultipleValue);
+
+                var patchConfig = c.Argument("patchConfig", "Path to the configuration file for patch update");
+
+                c.HelpOption("-h|--help");
+
+                c.OnExecute(() => OnUpdatePatch(pathOption, updatePatchConfigOption, updatePackageOption, updateRepoOption, patchConfig));
```

Comment:
Code organization: would be nicer to move the executed methods in to separate types and simply pass in the values to the method:

```
public static class UpdatePatch
{
   public void Execute(UpdatePatchOptions options)
   {
     ...
   }
}
```